packages/embed/src/subjects.ts

```diff
@@ -39,7 +39,7 @@ export const createSubjectManager = () => {
     applePayAuthorized$: createSubject<ApplePayJS.ApplePayPaymentToken>(),
     appleCompletePayment$: createSubject<boolean>(),
     appleAbortSession$: createSubject(),
-    appleSessionError$: createSubject(),
+    appleSessionError$: createSubject<string>(),
```

Comment:
@douglaseggleton is this something a merchant can listen to? If so, does this run the risk of anything breaking of they already listen to this event?

Reply:
This is an internal event. It will go to Embed UI.

Reply:
Tnx. Have a great trip!